apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -62,6 +64,29 @@ var styles = {
     height: HEADER_HEIGHT,
     lineHeight: HEADER_HEIGHT + 'px'
   },
+  helpTab: {
+    float: 'left',
+    paddingTop: "6px",
+    paddingLeft: "30px",
+
+  },
+  tab: {
+    paddingLeft: "10px",
+    marginRight: "5px",
+    paddingRight: "10px",
+    fontWeight: "bold",
+    color: color.charcoal,
+    paddingBottom: "6px",
+  },
+  highlighted: {
+    paddingLeft: "10px",
+    marginRight: "5px",
+    paddingRight: "10px",
+    fontWeight: "bold",
+    paddingBottom: "6px",
+    color: "#333",
+    borderBottom: "2px solid #333"
```

Comment:
we should be trying to only use colors from our palette as much as possible (i.e. avoid #333 if we can)

Reply:
nit: in general, i prefer to group all attributes of the same type near each other (i.e. have all the padding attributes, then all the margin attributes)